### PR TITLE
Changed ActiveCamera to have Option inside of it

### PR DESCRIPTION
**Pull request by jojolepro**
Using `Option<Read<ActiveCamera>>` is limiting some use cases I'm experimenting with. This PR fixes that.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +29,7 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 [#1237]: https://github.com/amethyst/amethyst/pull/1237
 [#1256]: https://github.com/amethyst/amethyst/pull/1256
 [#1267]: https://github.com/amethyst/amethyst/pull/1267
+[#1277]: https://github.com/amethyst/amethyst/pull/1280
```
Comment (azriel91) [inline docs/CHANGELOG.md]: eh almost

Comment (jojolepro) [reply docs/CHANGELOG.md]: :(

**Comment by jojolepro** [comment]
Thanks for reviewing <3

**Comment by jojolepro** [comment]
Motive being, if you have a game with a single Camera and no ActiveCamera resource, you can't insert the ActiveCamera resource from a System.

That means you can't have self-contained systems creating a camera reliably. This also means that such system couldn't remove the added camera to get back to the single camera setup, because you can't remove the ActiveCamera resource either.

**Comment by jojolepro** [comment]
This: https://www.youtube.com/watch?v=5PSj79FmoDk

**Comment by Xaeroxe** [comment]
Sufficient for me, thanks!

**Comment by Xaeroxe** [comment]
Let's try this again

bors r+

**Pull request merged**